README.md

```diff
@@ -7,3 +7,5 @@ Clone the repo and cd into the repo directory and then run `yarn`
 This a work in progres and lack some styling. The major part of `Data-aRT` is fully implemented here is what it looks like (in simple wireframe design):
 
 ![](exxample.png)
+
+![](Capture.PNG)
```

Comment:
remove `![](capture.PNG)` from here add it to your pull request message

Reply:
done